src/modules/graphql/schemas/timeslot.gql

```diff
@@ -1,13 +1,15 @@
 # A type describing a single time slot on the class/student schedule
 type TimeSlot {
-    # 2 numeric chars (00, 01...) represents the [hour, day] of timeslot 
+    # 2 numeric chars (00, 01...) represents the [hour, day] of timeslot
```

Comment:
These are now `0_0` separated by underscore.
I'm approving this PR, but can you please update this comment just before you merge it?

Reply:
Hi @dimkinv ,
Done